js/piwik.js

```diff
@@ -1068,10 +1068,6 @@ if (typeof window.Piwik !== 'object') {
             /* performance timing */
             performanceAlias = windowAlias.performance || windowAlias.mozPerformance || windowAlias.msPerformance || windowAlias.webkitPerformance,
 
-            /* DOM Ready */
-            hasLoaded = false,
-            registeredOnLoadHandlers = [],
```

Comment:
registeredOnLoadHandlers + hasLoaded was only used for enableLinkTracking.

enableLinkTracking now uses existing `trackCallbackOnReady` instead